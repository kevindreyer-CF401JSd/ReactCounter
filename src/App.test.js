import React from 'react'
import App from './app'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

describe('<App />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('is alive at application start', () => {
    const component = shallow(<App />)
    expect(component.find('div')).toExist()
  })
  describe('For Up and Down events', () => {
    it('Assert state changes properly', () => {

    })
    it('Assert that state is being transferred to the DOM', () => {

    })
    it('Assert DOM stability via snapshot testing', () => {

    })
  })
})