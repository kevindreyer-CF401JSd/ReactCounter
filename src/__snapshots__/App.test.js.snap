// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div
  classname="container"
>
  <header>
    <h1>
      Kevin's React Counter App
    </h1>
  </header>
  <div
    className="counter"
  >
    <span
      className="down clicker"
      onClick={[Function]}
    >
       - 
    </span>
    <span
      className="count "
    >
      0
    </span>
    <span
      className="up clicker"
      onClick={[Function]}
    >
       + 
    </span>
  </div>
  <footer>
    Â© 2020 | Kevin Dreyer | Code Fellows
  </footer>
</div>
`;
